### ISSUE TITLE
Armor is calculated wrongly

### ISSUE THREAD

**Issue by Lavaeolous** [Lavaeolous/PF1-StatBlock-Converter-Module#610]
The armor values in the preview are calculated wrongly, see 
![image](https://user-images.githubusercontent.com/14833806/199067421-c5da7e76-cd0f-4557-a814-760315ec8b2b.png)

**Comment by Lavaeolous** [comment]
```
Vampire Shield Bearer

CR 12

XP 19,200

Human Vampire ranger 11 (Pathfinder RPG Bestiary 270)

NE Medium undead (augmented humanoid, human)

Init +7; Senses darkvision 60 ft.; Perception +31

DEFENSE

AC 33, touch 14, flat-footed 29 (+4 armor, +3 Dex, +1 dodge, +8 natural, +7 shield)

hp 131 (11d10+66); fast healing 5

Fort +13, Ref +14, Will +12

Defensive Abilities channel resistance +4, evasion; DR 10/magic and silver; Immune undead traits; Resist cold 10, electricity 10

Weaknesses vampire weaknesses

OFFENSE

Spd 40 ft.

Melee +3 heavy shield bash +21/+16/+11 (1d6+10), +2 slam +15 (1d4+5 plus energy drain) or +2 slam +20 (1d4+12 plus energy drain)

Ranged mwk throwing axe +15 (1d6+7)

Space 5 ft.; Reach 5 ft.

Special Attacks blood drain, children of the night, combat style (weapon and shield APG), create spawn, dominate (DC 19), energy drain (2 levels, DC 19), favored enemies (animals +4, good outsiders +2, humans +4), quarry

Ranger Spells Prepared (CL 8th; concentration +13)
3rd— greater magic fang, instant enemy
2nd—aspect of the bear , owl's wisdom
1st—alarm, longstrider, resist energy

TACTICS

Before Combat The vampires cast aspect of the bear, owl's wisdom, greater magic fang, and longstrider on themselves.

During Combat The vampires protect Lyanthari, using Spring Attack to lunge at the PCs and Shield Slam to push them away before retreating back to Lyanthari's side.

Morale The shield bearers protect their liege at all costs and consider death by her side an honor.

Base Statistics Without their spells, the vampires' base stats are AC 31, touch 14, flat-footed 27; Will +10; Speed 30 ft.; Melee +3 heavy shield bash +21/+16/+11 (1d6+10), slam +13 (1d4+3 plus energy drain) or slam +18 (1d4+10 plus energy drain); concentration +11; Wis 16; CMB +18; Skills Perception +29, Sense Motive +13, Survival +17.

STATISTICS

Str 24, Dex 16, Con -, Int 10, Wis 20, Cha 18

Base Atk +11; CMB +20; CMD 32

Feats Alertness, Blind-Fight, Combat Reflexes, Dodge, Endurance, Greater Shield Focus, Improved Initiative, Improved Shield Bash, Iron Will, Lightning Reflexes, Missile Shield , Mobility, Shield Focus, Shield Master, Shield Slam, Spring Attack, Toughness

Skills Bluff +12, Climb +11, Intimidate +18, Knowledge (dungeoneering) +14, Knowledge (geography, nature) +7, Perception +31, Sense Motive +15, Spellcraft +14, Stealth +23, Survival +19; Racial Modifiers +8 Bluff, +8 Perception, +8 Sense Motive, +8 Stealth

Languages Common

SQ change shape (dire bat or wolf, beast shape II), favored terrains (plains +2, underground +4), gaseous form, hunter's bond (companions), shadowless, spider climb, swift tracker, track +5, wild empathy +15, woodland stride

Gear mwk chain shirt, +3 heavy steel shield with shield spikes
```

**Comment by Lavaeolous** [comment]
Fixed in 4.0.3
